app/db/models.py

```diff
@@ -89,7 +89,7 @@ class DBPrivateAIKey(Base):
 
     def to_dict(self):
         return {
-            "id": self.id,
+            "id": self.id or -1,
```

Comment:
this magic `-1`  is a bit opaque. could make it a meaningful constant name or add a comment about what it means?